MagicalCryptoWallet/Services/WalletService.cs

```diff
@@ -510,35 +539,16 @@ private void ProcessTransaction(SmartTransaction tx, List<HdPubKey> keys = null)
 			Logger.LogInfo<WalletService>("Calculating dynamic transaction fee...");
 			Money feePerBytes = null;
 			using (var torClient = new TorHttpClient(IndexDownloader.Client.DestinationUri, IndexDownloader.Client.TorSocks5EndPoint, isolateStream: true))
-			using (var response = await torClient.SendAsync(HttpMethod.Get, $"/api/v1/btc/blockchain/fees/{feeTarget}"))
+			using (var response = await torClient.SendAndRetryAsync(HttpMethod.Get, $"/api/v1/btc/blockchain/fees/{feeTarget}"))
 			{
-				if (response.StatusCode != HttpStatusCode.OK) // Try again.
+				if (!response.IsSuccessStatusCode)
```

Comment:
@lontivero Note, we cannot use this, it is implemented as `return ((int)statusCode >= 200) && ((int)statusCode <= 299);` while we are specifically looking for `200 OK`. (I'm fixing it and all other occurrences in the code.)